src/requirements.txt

```diff
@@ -12,17 +11,13 @@ google-cloud-profiler==1.0.3
 google-cloud-storage==1.28.1
 grpcio==1.28.1
 httplib2==0.18.0
-hyperlink==19.0.0
 lxml==4.5.0
 mozprocess==1.1.0
 oauth2client==4.1.3
 protobuf==3.11.3
-python-dateutil==2.8.1
-pytz==2018.5
 PyYAML==5.1
 redis==3.3.11
 requests==2.21.0
-selenium==3.141.0
 six==1.14.0
 tlslite-ng==0.7.5
```

Comment:
tlslite-ng, twisted, and lxml are depended on by peach. This is not obvious at all, since the peach code is a zip in the resources dir. 

@mbarbella-chromium would it be possible to move these dependencies into that zip instead of having them here?

Reply:
I think I'd rather remove the zip. I'll file a bug for this.